cli/coverage_cmd.go

```diff
@@ -196,3 +212,63 @@ func divideSentinel(x, y, sentinel float64) float64 {
 	}
 	return q
 }
+
+// numLines counts number of lines that are
+// - not blank
+// - do not look like comment
+// in the given data
+func numLines(data []byte) int {
+
+	len := len(data)
+	if len == 0 {
+		return 0
+	}
+
+	count := 1
+	start := 0
+
+	comment := []byte{'/', '/'}
```

Comment:
Optional style suggestion, this might be nicer/more readable to write using this form:

```Go
comment := []byte("//")
```

End result is identical.

Also, if you're calling this `numLines` func a lot, maybe it's worth it to move `comment` declaration out so it's not allocated each time this runs. (But maybe it's not worth it.)